Classes/Network/PonyDebugger/FLEXNetworkObserver.m

```diff
@@ -1591,15 +1591,17 @@ typedef void (^SendMessageBlock)(
         [FLEXNetworkObserver.sharedObserver
             websocketTask:slf sendMessagage:message
         ];
-        completion = ^(NSError *error) {
+        
+        id completionHook = ^(NSError *error) {
             [FLEXNetworkObserver.sharedObserver
                 websocketTaskMessageSendCompletion:message
                 error:error
             ];
+            completion(error);
```

Comment:
`completion` is nullable, this needs to check for that before calling it or it will crash